openflow13/openflow13.go

```diff
@@ -106,15 +107,36 @@ func Parse(b []byte) (message util.Message, err error) {
 	case Type_Error:
 		message = new(ErrorMsg)
 		err = message.UnmarshalBinary(b)
+		if err != nil {
+			return
+		}
+		errMsg, _ := message.(*ErrorMsg)
+		if errMsg.Type == ET_EXPERIMENTER {
+			message = new(BundleError)
+			err = message.UnmarshalBinary(b)
```

Comment:
Please make the same change here too and then we should be good to go.

Reply:
Thanks for review, I have updated as the comments.